pkgs/applications/video/obs-studio/plugins/obs-ndi.nix

```
@@ -37,8 +37,11 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Network A/V plugin for OBS Studio";
     homepage = "https://github.com/Palakis/obs-ndi";
-    maintainers = with maintainers; [ jshcmpbll ];
     license = licenses.gpl2;
-    platforms = with platforms; linux;
+    maintainers = with maintainers; [ jshcmpbll ];
+    platforms = platforms.linux;
+    hydraPlatforms = []; # "ndi" cannot be built at hydra. Ideally,
```

Maybe you can inherit it by hand then?
```suggestion
    hydraPlatforms = ndi.meta.hydraPlatforms; # "ndi" cannot be built at hydra. Ideally,
```